app/workers/new_page_worker.rb

```diff
@@ -0,0 +1,29 @@
+class NewPageWorker
+  include Sidekiq::Worker
+
+  def perform(page_id)
+    page = Page.find(page_id)
+    make_screenshot(page)
+    parse_html_content(page)
+    puts "Page #{page.id} created!"
```

Comment:
change to 
Rails.logger.info "Page #{page.id} processed!"
because page was already created